pkg/kv/kvserver/client_lease_test.go

```diff
@@ -1537,79 +1536,6 @@ func TestLeaseTransfersUseExpirationLeasesAndBumpToEpochBasedOnes(t *testing.T)
 	require.Equal(t, roachpb.LeaseExpiration, mu.lease.Type())
 }
 
-// TestLeaseUpgradeVersionGate tests the version gating for the lease-upgrade
```

Comment:
Why are we deleting this test? Also, can you move the delete into a separate commit?

Reply:
Sure, deleted in #111396